File: Python Programming/problems on pattern printing/problem33.py
'''
33. Accept number from user and display below pattern. 
	  Input :  5  
	 Output : A B C D E 

	Author : Milind Nalawade
	
	Date: 29/06/2021
	
	language : java 14.0.2
'''
#====
#Logic
#====
def Print(no):
	i=0
	while no!=0:
	#chr convert interger to character and ord convert character
	#to integer (chr(ord(65)+1)) -> char(65) ='A'
	
		print(chr(ord('A')+i),end=" " )
		i+=1
		no=no-1
		

#============
#Entry point function
#============
def main():
	no=int(input("Enter your number : "))
	Print(no)

#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C++ Programming/problem on pattern printing/problem33.cpp
/*

33. Accept number from user and display below pattern. 
	  Input :  5  
	 Output : A B C D E 

	Author : Milind Nalawade
	
	Date: 29/06/2021
	
	language : C++ programming

*/
using namespace std;
#include<iostream>

//Logic

void Print(int no)
{
	char ch='A';
	while(no>0)
	{
		cout<<ch<<" ";
		ch++;
		no--;
	}
}

//Entry point function 

int main()
{
	int no=0;
	cout<<"Enter your number : ";
	cin>>no;
	Print(no);
	return 0;
}